python/lsst/ap/association/packageAlerts.py

```diff
@@ -405,6 +409,15 @@ def produceAlerts(self, alerts, visit, detector):
                         f.write(alertBytes)
 
         self.producer.flush()
+        visit_midpoint = Time(alert["diaSource"]["midpointMjdTai"], format='mjd', scale='utc')
+        produce_end_timestamp = int(time.time())  # Current time in seconds
+        total_time = produce_end_timestamp - visit_midpoint.unix_tai
```

Comment:
Since we compute the `total_time` (or other name, below) metric, we should add a `self.log.info` level log message to report it during processing as well.